ietf-spice-charter.md

```diff
@@ -1,14 +1,14 @@
 # Introduction
 
-A digital credential expresses claims, assertions, or attributes about a subject, such as their name or age, and their cryptographic keys. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
+A digital credential links claims about a subject and their cryptographic keys. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
```

Comment:
This change makes this sentence inaccurate. 
Perhaps this would be better:
```suggestion
A digital credential expresses claims about a subject. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
```

Reply:
Thanks! But this would loose the 'cryptographic keys'? Maybe 'links' does not work that well by itself... Hiow about:

```suggestion
A digital credential expresses claims about a subject and links them with cryptographic keys. Some sets of claim names have already been defined by the IETF and other standards development groups (e.g., OpenID Foundation).
```

Reply:
That works for me